app/misc/easemob.py

```diff
@@ -0,0 +1,95 @@
+# coding=utf-8
+
+import requests
+import json
+from requests.auth import AuthBase
+
+
+class Easemob(object):
+    """SDK for Easemob IM system"""
+    JSON_HEADER = {'content-type': 'application/json'}
+
+    def __init__(self, app):
+        self.app_id = app.config['EASEMOB_APP_ID']
```

Comment:
'EASEMOB_APP_ID'? is the "'" necessary? not sure if Python's syntax has changed. I could be wrong. Same for the following a few lines.